### PR TITLE
saving color mode

### PR DESCRIPTION
**Pull request by aloofbit**
Hi I wanted to add feature. You change the dark/light mode and when reload the page it stays!

reload-save-color

image

## html
- added "made with heart"

## css
- put the themes in variables this made it easier to change
- put a transition on more so it is not so sharp

# overall
- move some files
image

- deleted all package.json stuff (probably in trouble for this)
    - no build or serve can drop html in browser now. but put serve in case.
- remove github and put in .nojekyl. can host off main branch :root now
- changed some to make download little less

**Before**: 
image

**After**: 
image

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var state = localStorage.getItem('toggle') === 'true' ? true : false
```
Comment (FichteFoll) [inline src/script.js]: Please use a more descriptive name like "darkMode" or "style" and then set it to "light" or "dark".

File: index.html [hunk 2]
```diff
@@ -1,110 +1,94 @@
...
-                        </div>
-                    </div>
-                </div>
-            </section>
-        </main>
-    </div>
-    <script src="dist/js/main.min.js"></script>
+         </div>
+      </div>
+      <div class="love-sublime">Made with ❤️ on Sublime Text</div>
```
Comment (FichteFoll) [inline index.html]: I find these "made with :heart:" kinda cringe and superfluous tbh, but I'm not strictly against it if @TerminalFi likes it. It just doesn't convey any useful information (like, surely you didn't make it with hate or something, but even if you did you probabaly wouldn't tell).

File: index.html [hunk 3]
```diff
@@ -1,110 +1,94 @@
...
+         </div>
+
+         <div class="card">
+            <div class="left">
+               <a href="http://discord.sublimetext.io/">
+                  <img class="app-icon" src="./src/images/tinified/discord.png" alt="discord logo">
+               </a>
+            </div>
+            <div class="right">
+               <a href="https://discord.sublimetext.io/">
```
Comment (FichteFoll) [inline index.html]:
```suggestion
               <a href="http://discord.sublimetext.io/">
```

https doesn't work with the redirect, unfortunately.

Comment (aloofbit) [reply index.html]: oops! i updated, sorry about that!

File: src/script.js (hunk 1 above)
Comment (aloofbit) [reply src/script.js]: ok i tried to improve on this! please see!

File: index.html (hunk 2 above)
Comment (aloofbit) [reply index.html]: ok ok i understand! but unfortunately i am only the developer and you are going to have to speak with my product manager :(

image


can we come compromise maybe? i remove html. i put in a console.log
image

**Comment by TerminalFi** [comment]
> remove github and put in .nojekyl. can host off main branch :root now

I’d prefer to keep everything hosted of off the gh-pages branch

**Comment by aloofbit** [comment]
> > remove github and put in .nojekyl. can host off main branch :root now
> 
> I’d prefer to keep everything hosted of off the gh-pages branch

Ok ok! I'm not very familiar with the right buttons to press for that. I put the gh_pages.yaml file back can you help me on the proper settings?

File: src/script.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+htmlToggle.onclick = () => onClickToggle()
+
+// initial state
+const readStorage = localStorage.getItem(UI_MODE_SAVE_KEY)
+const initialState = readStorage ?? UI_LIGHT
+initialize(initialState)
+
+function initialize(state) {
+   console.log('Made with ❤️ in Sublime Text')
+   const toggle = state === UI_DARK ? true : false
```
Comment (FichteFoll) [inline src/script.js]: ` const toggle = state === UI_DARK`

(github didn't allow me to make a suggestion for some reason)

**Review by FichteFoll — CHANGES_REQUESTED** [review]

File: src/script.js (hunk 4 above)
Comment (aloofbit) [reply src/script.js]: ooo that is tidy 💪 i updated!

**Comment by FichteFoll** [comment]
Looks good to me. I made some small adjustments. 

Pipeline should also work as-is.

**Review by FichteFoll — APPROVED** [review]

**Pull request merged**